src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-move-element.js

```diff
@@ -0,0 +1,87 @@
+(function (global) {
+    let moveUpCommand, moveDownCommand;
+
+    if (CKEDITOR.plugins.get('ezmoveelement')) {
+        return;
+    }
+
+    function fireEditorInteraction(editor, evt, target) {
+        const e = {
+            editor: editor,
+            target: target.$,
+            name: evt,
+        };
+
+        editor.fire('editorInteraction', {
+            nativeEvent: e,
+            selectionData: editor.getSelectionData(),
+        });
+    }
+
+    moveUpCommand = {
+        exec: function (editor, data) {
+            let focused = editor.elementPath().block,
```

Comment:
Each variable should be declared separately. We don't follow the "single-var" rule anymore.